README.md

```diff
@@ -1,3 +1,23 @@
-# RustBridge in a Box
+# in-a-box
 
-Base resources and guidelines for organizing and mentoring RustBridge sessions
+`in-a-box` is the best way to kickstart a RustBridge event! This repository includes all kinds of interesting and valuable content for future and current event organizers to use. The repository is broadly divided into four sections:
+
+### Lesson Plans
+
+Every RustBridge event is different, and therefore we provide different lesson plans (also called Agendas) for different requirements. Some of them take longer and are more complex, while others are suited for just a 2-hour event with mostly newcomers to the language.
+
+### Logistics
+
+This includes helpful information for making the technical and often times nasty details about organizing an event as easy and smooth as possible, for example things like finding a venue, organizing food and drinks, or managing people.
+
+### Outreach
+
+Often times kicking things off is the hardest part of organizing an event. We had a lot of people asking us how they should approach potentially interested parties, so we wrote copy-pasteable content you can use.
+
+### Resources
+
+These are resources that don't particularily fit in any other category, but are for general use (and might not be specific to RustBridge)
+
+## Contributing
+
+Running an event? You've discovered new groundbreaking information that might change the way we think about learning Rust forever? You can open a Pull Request with your changes or additions here! There's no specific guidelines as to how it should be formatted other than being Markdown.
```

Comment:
this line sucks

Reply:
How about: "Have you used these materials or run your own workshop? We'd love to learn and grow from your experience. Please feel free to file any issues with questions or comments, or open up a PR with new materials!"